routes/flock.json

```diff
@@ -119,43 +119,282 @@
     ]
   },
   {
     "name": "Goose V",
-    "thumbnail": "/images/flock/goosex.png",
+    "thumbnail": "/images/flock/goose5.png",
     "url": "goose_v",
+    "desc": "UnLIMited potential.",
+    "banner": "/images/flock/goosex/torontorender.jpg",
+    "sections": [
+      {
+        "title": "Specifications",
+        "content": [
+          "GOOSE V is the result of refinining and optimizing GOOSE IV. We decided to expand upon the feasibility of Linear Induction Motors by incorporating it alongside a conservative, scalable and cost-effective subsystem design approach. More specifically, our team has gained tremendous insights from testing a small-scale LIM prototype, a high speed flywheel braking rig and a physial battery failure scenario.",
+          "The prototype is about 0.533m (21.68\") wide, 2.296m (7.5') long, 0.356m (14.76\") tall, around 82 kg in weight, and is designed to reach a safe and faster speed of 44m/s."
+        ]
+      },
+      {
+        "title": "Linear Induction Motor",
+        "content": [
+          "Like a Rotary Induction Motor but cut up on its centre axis and with the stator and rotor unrolled flat, a linear induction motor produces thrust through a linear force rather than through torque. This allows it to function on an unpowered conductive track, thereby saving costs in maintenance and per-unit track costs. Despite the disadvantage of end effects (or \"turbulence\") leading to lower speeds, the LIM offers contactless propulsion on a simpler track such as an aluminum one. Furthermore, the LIM allows for a centralized design that reduces the weight of the entire pod, and there are a variety of opportunities to scale up the LIM design and promises of better levitation technologies to use with the LIM. We use the DLIM (or Double-sided LIM) design, which involves 2 LIMs on either side of the I-beam track, to allow for a stronger magnetic field to be maintained."
+        ],
+        "parallax": "/images/flock/goose5/DLIM_Render.png"
+      },
+      {
+        "title": "Frame",
+        "content": [
+          "The main structural component of the pod, the frame houses all of the pod's mechanical and electrical subsytems. Made from 6061-T6 square aluminum tubing, the frame is welded in a truss structure and weighs 10 kg. The material and construction of the frame is designed to withstand all acceleration, braking, vibration and structural loads during the run."
+        ],
+        "parallax": "/images/flock/goose5/FRAME_DESIGN .JPG"
+      },
+      {
+        "title": "Guidance",
+        "content": [
+          "The guidance system maintains lateral and horizontal stability. It accounts for any variations or imperfections in the track that may induce vibrations and bump impulses to the vehicle, and ensures the pod stays on track without damage. The guidance system also ensures that the pod maintains consistent contact with the track, as the LIM and sensors are affected by even the smallest of deviations.",
```

Comment:
Change "and ensures the pod stays on track without damage"

to "ensuring the pod stays on track without damage"